setup.py

```diff
@@ -45,6 +45,8 @@
         'console_scripts': [
             'pom_battle=pommerman.cli.run_battle:main',
             'pom_tf_battle=pommerman.cli.train_with_tensorforce:main',
+            'ioncl=pommerman.network.ion_client:init',
```

Comment:
`ion_client`
`ion_server`

Reply:
`ionse` is ambiguous and fairly silly looking. `ioncl` isn't far from that.
a further reason to do this is to match it up with the actual functionality in the codebase, which now (correctly) has `ion_client` as the module.

Reply:
Fine but I feel this is a BDFL move.